tests/graph_stores/test_sparql.py

```diff
@@ -0,0 +1,55 @@
+
+import pytest
+from llama_index.graph_stores.sparql import SparqlGraphStore
```

Comment:
We will likely need to skip these tests if the dependency wrapper for sparql is not installed. Check out the tests for other graph stores as an example